app/experimenter/base/tests/test_load_dummy_experiments.py

```diff
@@ -1,8 +1,9 @@
 from django.core.management import call_command
 from django.test import TestCase
 
-from experimenter.experiments.models import Experiment, NimbusExperiment
+from experimenter.experiments.models import NimbusExperiment
 from experimenter.experiments.tests.factories import NimbusExperimentFactory
+from experimenter.legacy.legacy_experiments.models import Experiment
```

Comment:
Looks like we might need to stop using `Experiment.STATUS_CHOICES` and replicate it on the nimbus side 🤔

Reply:
Good catch!  I'm finding a couple little weird overlap things, like there was bucketing defined in Legacy that wasn't being used for anything at all.  So hopefully we'll find more of these kinds of things as we continue this process 🙏

Reply:
Do we have any issues filed for these lil overlaps? Do you think that should be part of this work, or can it come later?

Reply:
I don't have individual issues filed for the overlaps/duplication, some were already handled as part of this PR, and I suspect that they'll have to be resolved as I move things around, so I'm thinking of them more as side effects of the changes rather than the focus of the changes themselves.  If I get through a whole pass on this and we still find some them we can try filing tickets for them more individually.

Reply:
That sounds good to me. I just want to make sure nothing falls through the cracks 😄